helm/examples/local/README.md

```diff
@@ -2,12 +2,12 @@
 
 ## Directory contents
 
-### hpcc-localfile/
+### localfile/
```

Comment:
should shouldn't be changed, the chart is called hpcc-localfile.
The only change required was what the user uses to name the instance of the install (so that it matches the example).